### PR TITLE
Removed default builder image name from `acr pack build`

### PR DESCRIPTION
**Pull request by dorfire**

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/acr/_constants.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 ACR_TASK_YAML_DEFAULT_NAME = 'acb.yaml'
 
-ORYX_PACK_BUILDER_IMAGE = 'mcr.microsoft.com/oryx/pack-builder:stable'
+ACR_CACHED_BUILDER_IMAGES = ('cloudfoundry/cnb:bionic',)
```
Comment (dorfire) [inline src/azure-cli/azure/cli/command_modules/acr/_constants.py]: @northtyphoon FYI

File: src/azure-cli/azure/cli/command_modules/acr/pack.py [hunk 2]
```diff
@@ -56,8 +56,8 @@ def acr_pack_build(cmd,  # pylint: disable=too-many-locals
     if platform_os != OS.linux.value.lower():
         raise CLIError('Building with Buildpacks is only supported on Linux.')
 
-    if builder != ORYX_PACK_BUILDER_IMAGE and not pull:
-        logger.warning('Using a non-default builder image; `--pull` is probably needed as well')
+    if builder not in ACR_CACHED_BUILDER_IMAGES and not pull:
```
Comment (dorfire) [inline src/azure-cli/azure/cli/command_modules/acr/pack.py]: @northtyphoon this will allow us to support more cached images in the future

**Comment by JasonRShaver** [comment]
@tjprescott We're holding to get this into 2.0.70 (code complete on 7/24); any concerns about getting this reviewed by then?

**Comment by yugangw-msft** [comment]
@northtyphoon, please review. I will merge once you approve it

**Pull request merged**